etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -18,7 +18,7 @@ stackhpc_firewalld_zones: |
   {% set _ = network_zones.append({'zone': network | net_zone }) %}
   {% endif %}
   {% endfor %}
-  {{ network_zones }}
+  {{ network_zones | unique }}
```

Comment:
Just for clarity, will the ```list``` filter also remove duplicates?
 Do I need to use it as well as ```unique``` or instead of?
 Should I apply the list filter to every instance of the "for loop" style variable definitions?

Reply:
I've slapped it on the two big output variables ```stackhpc_firewalld_rules``` and ```stackhpc_firewalld_zones```, which I _think_ will do it. The loops have been behaving fine so far, but if you think I should add it inside them I can do that too